src/hooks/siteDashboardHooks/useGetSiteLaunchStatus.ts

```diff
@@ -0,0 +1,20 @@
+import { UseQueryResult, useQuery } from "react-query"
+
+import { SITE_DASHBOARD_LAUNCH_STATUS_KEY } from "constants/queryKeys"
+
+import * as SiteDashboardService from "services/SiteDashboardService"
+
+import { SiteLaunchDto } from "types/siteLaunch"
+
+export const useGetSiteLaunchStatus = (
+  siteName: string
+): UseQueryResult<SiteLaunchDto> => {
+  return useQuery<SiteLaunchDto>(
+    [SITE_DASHBOARD_LAUNCH_STATUS_KEY, siteName],
+    () => SiteDashboardService.getSiteLaunchStatus(siteName),
+    {
+      retry: false,
+      refetchOnWindowFocus: false,
```

Comment:
nit: i think both of these can be set on the `queryClient` directly, saving us the trouble of repeating it.

separately, if this fetches data from db (and it should, right?) then `refetch: true` should be preferred as it won't cost us any gh api calls

Reply:
hey, yup the `refetechOnWindow` is redundant. However, since this query client is a central client used in multiple parts of this codebase, I dont want to increase the surface area of possible bugs in this PR! 

I am assuming you are referring to `refetchOnWindowFocus` ya. For now, the part of the state of the site launch process (ie, the current checklist number) is currently held in the frontend, so I want to be more careful on when we refetch this! (ie, the frontend should invalidate the query intentioanlly when the user is ready to launch the site and completed all the tasks in the checklist)

Reply:
sure